web/src/routes/backOffice/employees/edit/EditEmployee.svelte

```diff
@@ -77,13 +77,24 @@
 				scheduleEnd,
 			},
 		});
-
 		if (res.error) {
-			toast.show({
-				type: "error",
-				title: $t("error.unexpected.title"),
-				description: $t("error.unexpected.description"),
-			});
+			switch (res.error.code) {
+				case "conflict":
+					toast.show({
+						type: "error",
+						title: $t("employees.username.taken.title"),
+						description: $t("employees.username.taken.description"),
```

Comment:
I think that we usually have a generic `conflict` title so it can be reused in the future
```suggestion
						title: $t("employees.update.conflict.title"),
						description: $t("employees.update.conflict.description"),
```